ipatests/test_xmlrpc/test_automember_plugin.py

```diff
@@ -34,6 +34,7 @@
 
 import pytest
 import unittest
+from lib389.utils import ds_is_older
```

Comment:
We have to backport the patch to IPA 4.7 but we cannot use ``lib389`` in 4.7 because the branch still supports Python 2 and ``lib389`` is not available for Python 2.

Also this will check the version that is installed on the test runner host, not the version on the host that executes the command. You can use ``ipaserver.install.replication.get_ds_version`` to fetch the version number from the Root DSE.